frontend/doc-site/04__Analyze Public Data/4_4__Annotate.mdx

```diff
@@ -15,31 +15,28 @@ After you successfully complete the automatic annotation workflow, you can explo
 - use this predictive annotation workflow with data generated by 10X platforms
 - use the tissue model which is most closely related to the tissue profiled in your single cell experiment (currently available models are: lung, pan-tissue immune)
 
-## Installation of CELLxGENE[annotate] (OS X)
-
+## Installation of CELLxGENE[annotate]
 
 You can install the latest experimental release of CELLxGENE Annotate to gain access to the automatic annotation subcommand. It is highly recommended to perform installation in a clean virtual environment using [Conda](https://docs.conda.io/en/latest/) or [venv](https://docs.python.org/3/library/venv.html) with Python version 3.9+. Find examples of virtual environment setup using venv and conda below.
 
-CELLxGENE[annotate] is currently only supported on OS X.
+CELLxGENE[annotate] is currently only tested on OS X and Ubuntu operating systems.
 
-### venv setup
+### virtualenv setup
 
-For an overview of `venv`, refer to the link in the above section. If you are on an apple silicon machine (i.e mac with M1/M2 processor), you must use conda as an environment manager (refer to the section directly below).
+For an overview of `venv`, refer to the link in the above section.
 
 ```
-brew install pyenv  # used by MLflow
```

Comment:
no longer needed, since we no longer instruct user to use `--conda-env-manager virtualenv`